questions/align-center-middle.question.md

```diff
@@ -32,3 +35,71 @@ class ButtonApp(App):
 if __name__ == "__main__":
     ButtonApp().run()
 ```
+
+If you use the above on multiple widgets, you'll find they appear to
+"left-align" in the center of the screen, like this:
```

Comment:
"Left align in the center of the screen" sounds strange.

Reply:
This is how folk have termed it on a couple of occasions when asking about it, so it's as much faqtory-bait as anything.